entry/src/main/resources/base/graphic/ic_launcher_background.xml

```diff
@@ -0,0 +1,170 @@
+<?xml version="1.0" encoding="utf-8"?>
+<vector xmlns:ohos="http://schemas.huawei.com/res/ohos"
```

Comment:
animation is not working as per original lib

Reply:
Animation is not working as per android lib due to some API mismatch in HOS. Right now the same is kept as open and working in parallelly